src/main/java/com/cleanroommc/groovyscript/compat/vanilla/Player.java

```diff
@@ -0,0 +1,21 @@
+package com.cleanroommc.groovyscript.compat.vanilla;
+
+import net.minecraft.item.ItemStack;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Player {
+
+    public static final String givenItems = "groovyscript_given_items";
```

Comment:
`GroovyScript:GivenItems` convention preferred

Reply:
Also, `public static final` variable names should normally be `CONSTANT_CASE`, `GIVEN_ITEMS` here